nmpolicy/operations.go

```diff
@@ -18,13 +18,11 @@ package nmpolicy
 
 import (
 	"fmt"
-	"time"
 
-	"github.com/nmstate/nmpolicy/nmpolicy/internal/capture"
-	"github.com/nmstate/nmpolicy/nmpolicy/internal/expander"
-	"github.com/nmstate/nmpolicy/nmpolicy/internal/lexer"
-	"github.com/nmstate/nmpolicy/nmpolicy/internal/parser"
-	"github.com/nmstate/nmpolicy/nmpolicy/internal/resolver"
+	yaml "sigs.k8s.io/yaml"
+
+	"github.com/nmstate/nmpolicy/nmpolicy/internal"
+	internaltypes "github.com/nmstate/nmpolicy/nmpolicy/internal/types"
```

Comment:
I would use the term `unmarshaled` instead of internal.

Reply:
They are not unmarshaled they are just the internal types we are using, apart from the fact that is better to be explicit with package naming at imports for example "core/v1" is corev1 or "meta/v1/" is metav1 this kind of things.